### ISSUE TITLE
Abnormal muscle morphology in electron microscopy

### ISSUE THREAD

**Issue by MLK6014** [obophenotype/human-phenotype-ontology#6372]
**Preferred term label:**
Abnormal muscle morphology in electron microscopy
**Synonyms**
Abnormal muscle morphology in EM

**Definition (free text, please give PubMed ID)**
Ultrastructural abnormalities in human skeletal muscle observed through electron microscopy.

**Parent term (use hpo.jax.org/app)**
Abnormal skeletal muscle morphology HP:0011805

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mitochondrial disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Hi @MLK6014  -- the HPO usually does not include the type of investigation in the term, with the exception being findings that are 100% specific to a given technology, such as EEG. While there are lots of findings that can only be seen with EM, there are also many findings that can also be appreciated on routine histology. Therefore, it would not be a good term and would lead to a lot of double parentage for items that can be seen by two methods.
Could I suggest instead that we develop terms for individual anomalies visible by EM? If possible add new individual term requests.
Thanks!
